src/sentry/snuba/referrer.py

```diff
@@ -892,6 +892,7 @@ class Referrer(StrEnum):
     REPLAYS_QUERY_QUERY_REPLAYS_DATASET_SUBQUERY = "replays.query.query_replays_dataset_subquery"
     REPLAYS_QUERY_BROWSE_SIMPLE_AGGREGATION = "replays.query.browse_simple_aggregation"
     REPLAYS_FILE_REFERRER = "replays.query.download_replay_segments"
+    REPLAY_SCRIPTS_DELETE_REPLAYS = "replays.scripts.delete_replays"
```

Comment:
Nit: all the rest of these are `REPLAYS`:

```suggestion
    REPLAYS_SCRIPTS_DELETE_REPLAYS = "replays.scripts.delete_replays"
```